Dockerfile

```diff
@@ -23,11 +22,10 @@ COPY internal/controller/ internal/controller/
 # by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
 RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager cmd/main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
-WORKDIR /
-COPY --from=builder /workspace/manager .
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9-1029
```

Comment:
What's the reason for using ubi8 and not 9?

Reply:
No good reason. We can use rhel 9.

Reply:
I changed the images to use rhel 9